File: OOP/tic-tac-toe-board-game-app/src/com/techlab/player/Player.java
package com.techlab.player;

import com.techlab.board.Mark;

public class Player {

	private String name;
	private Mark mark;

	public Player(String name, Mark mark) {
		if (name.equals(null)&&mark.equals(null))
			throw new NullPointerException();
		this.name = name;
		this.mark = mark;
	}

	public String getName() {
		return name;
	}

	public Mark getMark() {
		return mark;
	}
}

File: nodeJs/tic-tac-toe-board-game/player.js
"use strict";
class Player {
    constructor() {
        this._name = null;
        this._mark = null;
    }
    set name(name) {
        this._name = name;
    }
    set mark(mark) {
        this._mark = mark;
    }
    get name() {
        return this._name;
    }
    get mark() {
        return this._mark;
    }
}
module.exports = Player;